src/data-model-providers/codegen/CodegenDataModelProvider.cpp

```diff
@@ -277,6 +286,18 @@ CHIP_ERROR CodegenDataModelProvider::Attributes(const ConcreteClusterPath & path
         ReturnErrorOnFailure(builder.Append(AttributeEntryFrom(path, attribute)));
     }
 
+    DataModel::AttributeEntry globalListEntry;
+
+    globalListEntry.readPrivilege = Access::Privilege::kView;
```

Comment:
Might be worth documenting that this is per-spec, and if this changes (e.g. a new global attribute with different access is added) we need to update this code?